README.md

```diff
@@ -7,6 +7,19 @@ that can be reused and kept up to date across various pieces of content.
 
 Use [GOV.UK Docker](https://github.com/alphagov/govuk-docker) to run any commands that follow.
 
+To run the application:
+
+```bash
+govuk-docker run content-block-manager-app
+```
+
+If you want to run the application with Whitehall, Publisher, Content Store and the frontend apps running (allowing
+you to test the whole publishing stack, as well as previewing content):
+
+```bash
+govuk-docker run content-block-manager-full
+```
+
```

Comment:
this seems to build all the resources. I've:

- created a block in CBM
- created a host doc in whitehall
- embedded my block in the host doc
- seen the host document listed in the "List of locations" in CDM

But when I try view the doc in front end (`http://www.dev.gov.uk/government/news/test`) I get a 503. Is that expected?